bindings/wasm/examples/src/explorer-did.js

```diff
@@ -0,0 +1,243 @@
+// Copyright 2020-2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import {
+    KeyPair,
+    KeyType,
+    Client,
+    Config,
+    Document,
+    MethodScope,
+    Service,
+    Timestamp,
+    VerificationMethod,
+    Network, ExplorerUrl
+} from '@iota/identity-wasm';
+
+/**
+ * This example shows a basic introduction on how to create a basic DID Document and upload it to the Tangle
```

Comment:
How does one actually run this? 

Typical examples are run by `npm run example:node -- <example_name>`. What do we execute here? Can you add the command to the comment or maybe a package.json task if appropriate? (Probably not appropriate since it's usually a one-off).